sourcegraph/cody
# Update One Box Code Snippets UI

closes: https://linear.app/sourcegraph/issue/SRCH-1100/one-box-search-improvements-correct-issues-from-dog-fooding
closes: https://linear.app/sourcegraph/issue/SRCH-1115/make-individual-search-results-in-one-box-collapsible

- Bring back the result hide functionality
- Include the ask Cody button at bottom of file
- Limit displayed results to 5, with show more button
- File path background should be opaque.

Designs: https://www.figma.com/design/eR495BJclVpdGdXf4lhSsH/One-box?node-id=632-7402&node-type=canvas&t=gs8ttscPyqpA2YTj-0

Loom: https://www.loom.com/share/1c1d01bc6f734eee8ff0f54cae8adf96

## Test plan

- Enable Onebox and execute code search intent
- Make sure only first 5 results are shown and "show more" and "ask the llm" button are shown at the bottom.
- Make sure the code snippets now have the dark bg same as the code snippets returned in the Cody response. 
- The file link does not have any border now. 
- Test the hide code snippet function by clicking on the caret left to the file link.

## Changelog
